public/app.js

```diff
@@ -216,5 +217,34 @@
         });
 
         $httpProvider.interceptors.push('jwtInterceptor');
-    }
+
+        $translateProvider
+            .useSanitizeValueStrategy('sanitizeParameters')
+            .useStaticFilesLoader({
+                prefix: 'static/languages-<%= VERSION %>/',
+                suffix: '.json'
+            })
+            .determinePreferredLanguage(function () {
+                let lang = navigator.userLanguage || navigator.language;
+
+                // if it is set via query, use that
+                const queries = document.location.search.substr(1).split('&');
+                for (query of queries) {
```

Comment:
reminder for future self: `x in y` loops over enumerable properties, `x of y` loops over iterable items

Reply:
but.... IE chokes on it
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility 

(I guess this would be the benefit of going with something like webpack, that it'd be easier to get polyfills and not worry about the fact that most of the useful bits in JavaScript are missing in IE)

Reply:
blegh, I'll do a standard loop.

Polyfills wouldn't fix this, it would need transpiling. Polyfills can't add syntax, only functions.